romancal/regtest/test_skycell_generation.py

```diff
@@ -13,12 +13,12 @@
 pytestmark = [pytest.mark.bigdata]
 
 EXPECTED_FILENAMES = [
-    "r512_p_v01001001001_r274dp63x31y80_f158_asn.json",
-    "r512_p_v01001001001_r274dp63x31y81_f158_asn.json",
-    "r512_p_v01001001001_r274dp63x32y82_f158_asn.json",
-    "r512_p_v01001001001_r274dp63x32y80_f158_asn.json",
-    "r512_p_v01001001001_r274dp63x32y81_f158_asn.json",
-    "r512_p_v01001001001_r274dp63x32y82_f158_asn.json",
+    "r00001_p_v01001001001_r274dp63x31y80_f158_asn.json",
+    "r00001_p_v01001001001_r274dp63x31y81_f158_asn.json",
+    "r00001_p_v01001001001_r274dp63x32y82_f158_asn.json",
+    "r00001_p_v01001001001_r274dp63x32y80_f158_asn.json",
+    "r00001_p_v01001001001_r274dp63x32y81_f158_asn.json",
+    "r00001_p_v01001001001_r274dp63x32y82_f158_asn.json",
```

Comment:
I moved the initial program string here to be r00001 to match the input exposures, which are program 00001.